<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForeignKeyForm</class>
 <widget class="QDialog" name="ForeignKeyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Foreign key editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblName">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtName"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lblTargetColumn">
       <property name="text">
        <string>Target column</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbTargetColumn"/>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="lblReferenceTable">
       <property name="text">
        <string>Reference table</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbReferenceTable"/>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="lblReferenceColumn">
       <property name="text">
        <string>Reference column</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbReferenceColumn"/>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="lblOnUpdateAction">
       <property name="text">
        <string>On UPDATE action</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbOnUpdateAction">
       <item>
        <property name="text">
         <string>NO ACTION</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RESTRICT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CASCADE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SET NULL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SET DEFAULT</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>On DELETE action</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbOnDeleteAction">
       <item>
        <property name="text">
         <string>NO ACTION</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RESTRICT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CASCADE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SET NULL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SET DEFAULT</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
